### ISSUE TITLE
Rotating log file

### ISSUE THREAD

**Issue by akusmin** [NLog/NLog#1662]
Hi,

I'm using NLog version 4.4.0-betaV15, dotnet 1.0.0-preview2-003131
On Linux environment logs are rotating by every entry. I've noticed that such behavior appears on setting parameter archiveEvery to "Day".

Test Target is:

```
<target name="file" autoFlush="true" xsi:type="File" fileName="${var:logdir:default=.}/${var:logname}.log" archiveFileName="${var:logdir:default=.}/${var:logname}.{####}.log" archiveEvery="Day"/>
```

Variables:

```
   <variable name="logdir" value="Log"/>
  <variable name="logname" value="server"/> 
```

There is no any useful information in a nlog-internal.log.

Regards.

**Comment by 304NotModified** [comment]
I think there is an unit test for this, but we having trouble to get the unit tests running on .net core. Could you help us on that?

**Comment by akusmin** [comment]
@304NotModified Sure. How can I help?

**Comment by 304NotModified** [comment]
Cool!

Any experience on running tests on AppVeyor/Travis?

We having the following issues:

- A stacktraceoverflowException when running tests on AppVeyor (probable wrong test, but dunno which)
- Even if the tests fails, AppVeyor reports a succes
- Running .NET Core / tests on Travis at all

**Comment by 304NotModified** [comment]
It seems Travis did run it, but lot of fails: https://travis-ci.org/NLog/NLog/jobs/161156448

**Comment by 304NotModified** [comment]
net35 runs, but doesn't report correctly: https://ci.appveyor.com/project/nlog/nlog/build/4.4.0-beta3154

The stackoverflowException seems to be in the EventLoggerTests

**Comment by snakefoot** [comment]
Maybe resolved with #1931 (Available with NLog 5.0 Beta6)

**Comment by 304NotModified** [comment]
AFAIK this is indeed solved with bets6.  :)
